osquery/tables/system/windows/users.cpp

```diff
@@ -24,81 +24,138 @@
 #include "osquery/core/conversions.h"
 
 namespace osquery {
+
+std::string psidToString(PSID sid);
```

Comment:
I don't think so, the current `process_ops` is cross-platform. I could roll something like that out though, as I know we've talked about having a `util` style header that's platform specific live somewhere. Any thoughts on where it'd best live?